README.md

```diff
@@ -11,10 +11,88 @@ Created by all the [contributors](https://github.com/qbicsoftware/sample-trackin
 A Liferay portlet to update the sample-tracking information of selected samples.
 
 ## Architecture
-![](interfaces_sample-tracking-update-portelt.png)
+![](readme-docs/figures/interfaces_sample-tracking-update-portlet.png)
 
 
 ## How to Install
 1. clone repository
 2. install maven
-2. run `mvn clean package`
+3. run `mvn clean package`
+
+## How to Use 
+
+This guide intends to showcase the features of the sample-tracking-update-portlet. 
+
+* [Select Samples](#select-samples):
+  * [Select Sample by ID](#select-sample-by-id)
+  * [Select Samples by File](#select-samples-by-file)
+* [Update Sample Information](#update-sample-information):
+    * [Update Sample Date](#update-sample-date)
+    * [Update Sample Location](#update-sample-location)
+    * [Update Sample Status](#update-sample-status)
+* [Remove Selected Samples](#remove-selected-samples)
+
+### Select Samples
+
+The sample-tracking-update-portlet currently offers two methods for adding the samples to update.  
+This can be achieved by either uploading a *.csv file containing the Sample Ids in the first column 
+or by providing the Sample IDs directly.   
+Details for both methods are outlined below. 
+
+#### Select Sample by ID
+The most convenient way of updating a singular sample is through the input of its sample code 
+into the "Sample ID" text field and pressing the "Add Sample Button". 
+![add](readme-docs/screenshots/AddSampleById.png)
+
+If the given sample ID is valid, the sample and its associated information will appear in the "Samples to update" grid 
+![add](readme-docs/screenshots/AddSampleByIdResult.png)
+
+Each subsequent Sample ID will be added to the "Samples to update" grid 
+![add](readme-docs/screenshots/AddSampleByIdAgain.png)
+ 
+#### Select Samples by File
+It is also possible to select the samples of interest by 
+uploading one or multiple *.csv files containing the sample codes in the first column.
```

Comment:
maybe add an example here on how the CSV needs to look like, kind of a style guide

Reply:
haha nvm found the example

Reply:
new question, why all the commas in the example csv?

Reply:
Good Catch! 👀 . 
Apparently Numbers exports csv with all columns even if they are empty. 
I updated the csv with only the set columns.